views/default-error.ejs

```diff
@@ -0,0 +1,24 @@
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title><%-data.name || data.message%></title>
+    <style><%- include style.css %></style>
```

Comment:
Looks like an earlier comment got lost - reposting.

I would prefer if we could use consistent whitespace formatting. Either always add spaces after opening/before closing EJS brackets, or never.

```
// no spaces
<%-data.name || data.message%>
<%-include styles/default-error.css%>

// spaces
<%- data.name || data.message %>
<%- include styles/default-error.css %>
```

I personally prefer the style with spaces.